layer/layer/canary/app.py

```diff
@@ -0,0 +1,97 @@
+import datetime
+import json
+import os
+from importlib.metadata import version
+
+import boto3
+
+from aws_lambda_powertools import Logger, Metrics, Tracer
+
+logger = Logger(service="version-track")
+tracer = Tracer()
+metrics = Metrics(namespace="powertools-layer-canary", service="PowertoolsLayerCanary")
+
+layer_arn = os.getenv("POWERTOOLS_LAYER_ARN")
+powertools_version = os.getenv("POWERTOOLS_VERSION")
+stage = os.getenv("LAYER_PIPELINE_STAGE")
+event_bus_arn = os.getenv("VERSION_TRACKING_EVENT_BUS_ARN")
+
+
+def handler(event):
+    logger.info("Running checks")
+    check_envs()
+    verify_powertools_version()
+    send_notification()
+    return True
+
+
+def on_event(event, context):
+    print(event)
+    request_type = event["RequestType"]
```

Comment:
```suggestion
@logger.inject_lambda_context(log_event=True)
def on_event(event, context):
    request_type = event["RequestType"]
```

No need for `print`. Logger can also inject context so we know Lambda request ID if something went wrong